src/PolyhedralGeometry/Polyhedron/standard_constructions.jl

```diff
@@ -1218,18 +1190,1083 @@ julia> nvertices(rand_subpolytope(cube(3), 5))
+    return Polyhedron{QQFieldElem}(pm_obj)
+end
+
+@doc raw"""
+   rss_associahedron(n::Int)
+
+Produce a polytope of constrained expansions in ambient dimension `n` according to 
+[RSS03](@cite)
+
+# Examples:
+To produce a $3$-dimensional associahedron in $5$-space, do: 
+```jldoctest
+julia> a= rss_associahedron(5)
+Polyhedron in ambient dimension 5
+
+julia> vertices(a)
+14-element SubObjectIterator{PointVector{QQFieldElem}}:
+ [0, 1, 12, 13, 16]
+ [0, 7, 8, 11, 16]
+ [0, 1, 4, 9, 16]
+ [0, 3, 4, 9, 16]
+ [0, 5, 6, 9, 16]
+ [0, 5, 8, 9, 16]
+ [0, 1, 8, 9, 16]
+ [0, 7, 10, 11, 16]
+ [0, 7, 12, 13, 16]
+ [0, 7, 12, 15, 16]
+ [0, 1, 12, 15, 16]
+ [0, 7, 8, 15, 16]
+ [0, 1, 4, 15, 16]
+ [0, 3, 4, 15, 16]
+
+julia> facets(a) 
+9-element SubObjectIterator{AffineHalfspace{QQFieldElem}} over the Halfspaces of R^5 described by:
+x₁ - x₂ ≦ -1
+x₁ - x₃ ≦ -4
+x₁ - x₄ ≦ -9
+x₂ - x₃ ≦ -1
+x₂ - x₄ ≦ -4
+x₂ - x₅ ≦ -9
+x₃ - x₄ ≦ -1
+x₃ - x₅ ≦ -4
+x₄ - x₅ ≦ -1
+```
+"""
+function rss_associahedron(n::Int)
+    @req n >= 2 "n must be at least 2."
+    return Polyhedron{QQFieldElem}(Polymake.polytope.rss_associahedron(n))
+end
+
+@doc raw"""
+    signed_permutahedron(d::Int)
+
+Produce the `d`-dimensional signed permutahedron. I.e. for all possible permutations of
+the vector $(1,\dots,d)$, all possible sign patterns define vertices of this polytope. 
+Contrary to the classical permutahedron, the signed permutahedron is full-dimensional. 
+
+# Examples:
+To produce the $2$-dimensional signed permutahedron, do: 
+```jldoctest
+julia> P = signed_permutahedron(2)
+Polyhedron in ambient dimension 2
+
+julia> vertices(P)
+8-element SubObjectIterator{PointVector{QQFieldElem}}:
+ [1, 2]
+ [-1, 2]
+ [1, -2]
+ [-1, -2]
+ [2, 1]
+ [-2, 1]
+ [2, -1]
+ [-2, -1]
+```
+"""
+function signed_permutahedron(d::Int)
+    @req d >= 1 "dimension >= 2 required"
+    @req d <= 8*sizeof(Int)-1 "dimension too high"
+    return Polyhedron{QQFieldElem}(Polymake.polytope.signed_permutahedron(d))
+end
+
+@doc raw"""
+    stable_set_polytope(G::Graph{Undirected}) 
+
+Produces the stable set polytope from an undirected graph `G`$=(V,E)$. 
+The stable set Polytope has the following inequalities: 
+$x_i + x_j \leq 1 \forall \{i,j\} \in E$,  
+$x_i \geq 0 \forall i \in V$ and
+$x_i \leq 1 \forall i \in V \text{ with } \mathrm{deg}(i)=0$ 
+
+# Example:
+The following produces first the standard cube in $3$ dimensions, and then 
+a bipyramid over the convex hull of the unit vectors. 
+```jldoctest
+julia> G = Graph{Undirected}(3)
+Graph{Undirected}(pm::graph::Graph<pm::graph::Undirected>
+{}
+{}
+{}
+)
+
+julia> S = stable_set_polytope(G)
+Polyhedron in ambient dimension 3
+
+julia> vertices(S)
+8-element SubObjectIterator{PointVector{QQFieldElem}}:
+ [1, 0, 0]
+ [1, 1, 0]
+ [1, 1, 1]
+ [1, 0, 1]
+ [0, 0, 1]
+ [0, 0, 0]
+ [0, 1, 0]
+ [0, 1, 1]
+
+julia> add_edge!(G, 1, 2)
+0
+
+julia> add_edge!(G, 1, 3)
+0
+
+julia> add_edge!(G, 2, 3)
+0
```

Comment:
```suggestion
julia> add_edge!(G, 1, 2);

julia> add_edge!(G, 1, 3);

julia> add_edge!(G, 2, 3);
```
Maybe we should remove the return value from `add_edge!`.